keyboards/sandortho/keymaps/default/keymap.c

```diff
@@ -0,0 +1,174 @@
+/* Copyright 2020 Luke LaCasse
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _RAISE,
+    _LOWER
+	//_EMPTY
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL,
+    EXCL,
+    AT,
+    POUND,
+    DOLLAR,
+    PERCENT,
+    CARAT,
+    AND,
+    STAR,
+    LPAR,
+    RPAR
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT(
+	KC_TAB,    KC_Q,     KC_W,    KC_E,    KC_R,       KC_T,      KC_Y,         KC_U,       KC_I,     KC_O,     KC_P,        KC_BSPACE,
+	KC_ESCAPE, KC_A,     KC_S,    KC_D,    KC_F,       KC_G,      KC_H,         KC_J,       KC_K,     KC_L,     KC_SCOLON,   KC_QUOTE,
+	KC_LSHIFT, KC_Z,     KC_X,    KC_C,    KC_V,       KC_B,      KC_N,         KC_M,       KC_COMMA, KC_DOT,   KC_SLASH,    KC_ENTER,
+    KC_NO,     KC_LCTRL, KC_LALT, KC_LGUI, MO(_LOWER), KC_NO,     KC_SPACE,     MO(_RAISE), KC_LEFT,  KC_DOWN,  KC_UP,       KC_RIGHT  //HC_NO before space is not a key in hardware
```

Comment:
Same for the rest of the layers, these should be updated to the layout macro
```suggestion
    KC_NO,     KC_LCTRL, KC_LALT, KC_LGUI, MO(_LOWER), KC_SPACE,     MO(_RAISE), KC_LEFT,  KC_DOWN,  KC_UP,       KC_RIGHT
```